pages/home.md

```diff
@@ -42,22 +42,18 @@ permalink: /
             </figure>
           </div>
           <div class="column is-offset-1 is-5">
-            <h2 class="title">All of your data, under your control</h2>
+            <h2 class="title">Pods: All your data, under your control</h2>
             <p class="content is-large">
             <a href="{{site.baseurl}}/about">Solid</a> lets people store their 
             data securely in decentralized data stores 
             called <b>Pods</b>. Pods are like secure personal web servers for data.
-            All data in a pod is accessible via the 
-            <a href="https://solid.github.io/specification/" target="_blank">Solid Protocol</a>.
-            When data is stored in someone's 
-            pod, they control who and what can access it.</p>
-                                
-            <p class="content is-large">
-            Solid is led by the inventor of the Web, 
-            <b><a href="{{site.baseurl}}/team">Sir Tim Berners-Lee</a></b>, 
-            to help realise his vision 
-            for its future.</p>
+            People control who and what can access data stored in their Pods.</p>
```

Comment:
"who and what" is unclear to me

Reply:
Updated to "who and what" to "persons and applications" (I also updated People to You since the section title is Your Pod)